src/System.Security.Cryptography.X509Certificates/tests/CollectionTests.cs

```diff
@@ -1438,6 +1441,18 @@ private static void TestExportStore(X509ContentType ct)
             }
         }
 
+        public static IEnumerable<object[]> StorageFlags
```

Comment:
Is it feasible\worthwhile to add a test that will verify the underlying key is not persisted at a lower level when using Ephemeral? Such as PersistKeySet flag will keep the cert around (until Disposed) and accessible by a different thread\code block while Ephemeral won't.